File: fizzbuzz.py
#!/usr/bin/env python
"""
http://c2.com/cgi/wiki?FizzBuzzTest
"""
import sys

for i in range(1, 101):
    if i % 3 == 0:
        sys.stdout.write("Fizz")
    if i % 5 == 0:
        sys.stdout.write("Buzz")
    if (i % 3 != 0) and (i % 5 != 0):
        sys.stdout.write(str(i))
    sys.stdout.write("\n")

File: fizzbuzz.c
#include <stdio.h>

int main(int argc, char ** argv) {
  int i = 1;
  for (int i=1; i <= 100; i++) {
    if (i % 3 == 0)
      printf("Fizz");
    if (i % 5 == 0)
      printf("Buzz");
    if (! ((i % 5 == 0) || (i % 3 == 0)))
      printf("%d", i);
    printf("\n");
  }
}
